src/scripts/modules/media/nav/nav.less

```diff
@@ -9,86 +9,115 @@
     background-color: @gray-lightest;
   }
 
-  > .book-nav {
-    .flexbox();
-    .flex-flow(row; nowrap);
-    .justify-content(center);
-    .user-select(none);
-
-    > .progress, > .back-to-top {
-      .flex(1; 1; 75%);
+  > .media-toolbar {
+    >* {
```

Comment:
I'm always a little skeptical whenever using the `*` in CSS.  Is there no better way to do this?